### ISSUE TITLE
pgpool cli, version, configs and production use?

### ISSUE THREAD

**Issue by junaid-a10** [CrunchyData/crunchy-containers#1258]
**Which example are you working with?** 
I am working with kube pgpool example.

**What is the current behavior?**
1) When I exec the pgpool pod and run `pgpool` command I get the following output:
```
bash-4.2$ pgpool 
2020-08-03 10:13:25: pid 54: WARNING:  could not open configuration file: "/etc/pgpool-II-12/pgpool.conf"
2020-08-03 10:13:25: pid 54: DETAIL:  using default configuration parameter values
2020-08-03 10:13:25: pid 55: FATAL:  could not open pid file as /var/run/pgpool/pgpool.pid. reason: No such file or directory
```

2) Some of the configs in pgpool do not map to latest pgpool version 4.1.2. What version of pgpool is being used in `crunchydata/crunchy-pgpool:centos7-12.3-4.3.2` image?

**What is the expected behavior?**
See the pgpool version or cli options.

**Other information** (e.g. detailed explanation, related issues,  etc)
I want to use pgpool in a production environment. Can the image provided by crunchy container suite be used for this purpose? Is there a recommended pgpool watchdog guide with crunchy container suite? 
Basically, I want to use pgPool for connection pooling and load balancing read write queries, do I need pgPool with Watchdog for high availability or should I just use Kubernetes deployment with say replication factor of n?

I have one other question regarding pgBouncer, Does Crunchy Postgres Operator provide HA for pgBouncer as well? I am assuming it does.

**Please tell us about your environment:**

* Operating System: CentOS 7.8.2003
* Where is this running ( Local , Cloud Provider) Local
* Storage being used (NFS, Hostpath, Gluster, etc): NFS
* Container Image Tag: crunchy-pgpool:centos7-12.3-4.3.2, Postgres Operator Version 4.3.2
* PostgreSQL Version: 12.3
* Platform (Docker, Kubernetes, OpenShift): Kubernetes
* Platform Version: Kubernetes version 1.18
